src/query/ast/tests/it/testdata/statement.txt

```diff
@@ -6265,3 +6265,299 @@ ShowGrantsOfShare(
+            quote: None,
+            span: Ident(16..17),
+        },
+        args: [],
+        source: Some(
+            Columns(
+                [
+                    ColumnDefinition {
+                        name: Identifier {
+                            name: "a",
+                            quote: None,
+                            span: Ident(36..37),
+                        },
+                        data_type: Int32,
+                        default_expr: None,
+                        comment: None,
+                    },
+                ],
+            ),
+        ),
+        as_query: Query {
+            span: [
+                SELECT(46..52),
+                Ident(53..54),
+                FROM(55..59),
+                Ident(60..61),
+            ],
+            with: None,
+            body: Select(
+                SelectStmt {
+                    span: [
+                        SELECT(46..52),
+                        Ident(53..54),
+                        FROM(55..59),
+                        Ident(60..61),
+                    ],
+                    distinct: false,
+                    select_list: [
+                        AliasedExpr {
+                            expr: ColumnRef {
+                                span: [
+                                    Ident(53..54),
+                                ],
+                                database: None,
+                                table: None,
+                                column: Identifier {
+                                    name: "a",
+                                    quote: None,
+                                    span: Ident(53..54),
+                                },
+                            },
+                            alias: None,
+                        },
+                    ],
+                    from: [
+                        Table {
+                            span: [
+                                Ident(60..61),
+                            ],
+                            catalog: None,
+                            database: None,
+                            table: Identifier {
+                                name: "t",
+                                quote: None,
+                                span: Ident(60..61),
+                            },
+                            alias: None,
+                            travel_point: None,
+                        },
+                    ],
+                    selection: None,
+                    group_by: [],
+                    having: None,
+                },
+            ),
+            order_by: [],
+            limit: [],
+            offset: None,
+            format: None,
+        },
+    },
+)
+
+
+---------- Input ----------
+CREATE FUNCTION t (a, b) RETURNS TABLE (a INT) AS select a from t where b=a and c=b;
+---------- Output ---------
+CREATE FUNCTION t (a, b) RETURNS TABLE (a Int32 NOT NULL) AS SELECT a FROM t WHERE b = a AND c = b
```

Comment:
`a, b` should have a data type?

Reply:
yes, will add. I already found this problem.